tests/test_cli.py

```diff
@@ -66,10 +66,25 @@ def test_percent(tmpdir, runner, create):
     assert '78%' in result.output
 
 
+def test_list_case_insensitive(tmpdir, runner, create):
+    result = runner.invoke(cli, ['list', 'default'])
+    assert not result.exception
+
+    result = runner.invoke(cli, ['list', 'DEfault'])
+    assert not result.exception
+
+    result = runner.invoke(cli, ['list', 'deFAUlT'])
+    assert not result.exception
```

Comment:
These tests look good. Can you also add one for the failure scenario (eg: lists `default` and `DEFAULT` both exist)?